bounties/npm/class-transformer/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-09-02",
+	"AffectedVersionRange": "*",
+	"Summary": "Prototype Pollution",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "class-transformer",
+		"URL": "https://www.npmjs.com/package/class-transformer",
+		"Downloads": "14759253"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-400",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "H",
+		"PR": "L",
+		"UI": "N",
+		"S": "U",
+		"C": "N",
+		"I": "L",
+		"A": "L",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "4.2"
+	},
+	"CVEs": [
+		"CVE-2020-7637"
+	],
+	"Repository": {
+		"URL": "https://github.com/typestack/class-transformer",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
+		"https://github.com/typestack/class-transformer/blob/a650d9f490573443f62508bc063b857bcd5e2525/src/ClassTransformer.ts#L29-L31"
```

Comment:
Change permalink to `develop` branch's codebase:

```
https://github.com/typestack/class-transformer/blob/develop/src/ClassTransformer.ts#L29-L43
```

Reply:
Updated, thanks 👍